packages/localizations/src/en-US.ts

```diff
@@ -802,6 +802,19 @@ export const enUS: LocalizationResource = {
     },
   },
   billing: {
+    start: {
+      headerTitle__billing: 'Plan & Billing',
+    },
+    manageBillingInfo: 'Manage billing info',
```

Comment:
How about create a parent with a key that will reference the name of the page simillar to what we are doing to `managePlanScreen` ?

Reply:
Done